src/Common/test-runtime/project.json

```diff
@@ -37,10 +37,10 @@
         "System.Linq": "4.4.0-beta-24911-02",
         "System.Linq.Parallel": "4.4.0-beta-24911-02",
         "System.Linq.Queryable": "4.4.0-beta-24911-02",
-        "System.Net.Http": "4.4.0-beta-24911-02",
+        "System.Net.Http": "4.4.0-preview1-25218-03",
```

Comment:
Don't need to change now. If we don't run our tests against NS1.3 anymore, I think we can remove this whole "netstandard1.3" section in the future.